### ISSUE TITLE
neural net

### ISSUE THREAD

**Issue by priyanga24** [awslabs/autogluon#608]
Hi,
my dataset has 800k rows with 600k:200k train and test split. it is a binary classificatio problem. I used neural network with hyper-parameter tuning with time limit set to **one hour**. But the process was running for more than 3 hours, so I stopped it. What is the issue?How can i solve it?

**Comment by aaronkl** [comment]
Hi, which scheduler/searcher did you use? How long is the training time of your neural network? Autogluon does not stop the evaluation of your algorithm if it runs over the time limit. So it might be that the first evaluation already takes 3 hours and Autogluon is effectively not doing anything. There are several things you could try: a) reduce the dataset, b) increase the time limit or c) use multiple workers and report the performance back to Autogluon after each epoch.

**Comment by jwmueller** [comment]
@priyanga24 
Please share the code you used and the output-logs.  Ideally, you will call `task.fit(..., verbosity=4)` to print detailed output-logs, and then share them here.  Do you have the same issue if you set `hyperparameter_tune=False`?

**Comment by Innixma** [comment]
Marking as resolved. Please re-open if the issue is still present and you have additional information to help us assist.
